wordpress-mobile/WordPress-iOS
# Image loader

Fixes #9250 

This PR introduce a new class `MediaLoader`. The intent is to facilitate loading images across the app. Specially those images that could be animated (gifs).

This PR also implements this `MediaLoader` in the `PostCardTableViewCell`, so it can be tested in action in the Posts list.

To test:

- Be sure to have plenty of images/gifs in your blog posts's feature images.
- Open the Blog list and check that all featured images are loading properly.
- Test in a WPcom sites, a self-hosted site and public/private posts.
